.changeset/odd-crabs-approve.md

```diff
@@ -0,0 +1,30 @@
+---
+'@itwin/itwinui-react': major
```

Comment:
Just double checking that this is the proper tag since this change is going in v3

Reply:
Yes it is, but you need to all the breaking changes more explicitly. For example, a lot of the props have been removed but they're not even mentioned in the changeset currently. At the same time, the changeset needs to be very concise because there are going to be _lots_ of bullet points in the changelog after we are done with all the work.

Reply:
This has been changed to mention the props that have been removed